### ISSUE TITLE
Custom CLAassistant comment wording

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [cla-assistant/cla-assistant#507]
_URL to the linked Repo/Org:_ https://github.com/Codecademy/console-fail-test/pull/53

_Steps to reproduce the problem:_

1. Create a new PR against a CLA-enabled organization
2. See the default text

_What is the expected result?_

As an organization, we'd like to have a consistent tone of wording in our online presence - including on GitHub. Not being able to customize the message makes that hard.

_What happens instead?_

> Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our Contributor License Agreement before we can accept your contribution.

Refiling of #105, but in 2019 instead of 2016. 😄

**Comment by ibakshay** [comment]
Hi @JoshuaKGoldberg, thank you for reaching out to us. We have to work on both UI and the backend to enable the Custom CLA Assistant wording, but right now we are  focusing on other high priority topics. 
We are in development of [CLA Assistant GitHub Action](https://github.com/cla-assistant/cla-assistant/issues/483) and we will definitely take  this feature (custom wording) into consideration and this action will be shipped sometime next year.  

However, you are very much welcome in raising a Pull Request to enable this feature if you would like to😄 . 
And  there are lot of companies who have customised the CLA Assistant solution and they self-hosted it. You can also customise the CLA Assistant in the way you want and you can self-host it. You can find the documentation [here](https://github.com/cla-assistant/cla-assistant/blob/master/README.md) on how to configure and self-host CLA Assistant.

**Comment by JoshuaKGoldberg** [comment]
Closing out my old issues for repos I no longer have context on. If anybody has a need for what this issue was asking about, I'd encourage them/you to file a new issue. Cheers!
